<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>666</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>781</width>
    <height>666</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000</width>
    <height>10000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>371</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_7" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="check_device">
         <property name="text">
          <string>Get device info</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Stream Format:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="yuv_stream">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>YUV420P</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="h264_stream">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>H.264</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="mjpeg_stream">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>MJPEG</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Stream Resolution:  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="outflow_mode"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_4" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>1st network model: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="model_1st"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_9" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>2nd network model:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="model_2nd">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_8">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_11" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Display Scaler:  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="scale_spinbox">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>25</height>
          </size>
         </property>
         <property name="minimum">
          <number>30</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="scale_value">
         <property name="minimum">
          <number>30</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_10" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Model Score:     </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="min_score_spinbox">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>25</height>
          </size>
         </property>
         <property name="minimum">
          <number>30</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="min_score_value">
         <property name="minimum">
          <number>30</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_5" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="load_fw_btn">
         <property name="text">
          <string>Start running model</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="two_net_model">
         <property name="text">
          <string>Start running models</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_6" native="true">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>0</y>
     <width>402</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="2">
     <widget class="roi_label" name="roi_setting_area">
      <property name="minimumSize">
       <size>
        <width>384</width>
        <height>216</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border:2px solid rgb(71, 169, 255);</string>
      </property>
      <property name="text">
       <string> Region of interest for model</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="Add_widget">
      <property name="text">
       <string>Add model</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="Del_widget">
      <property name="text">
       <string>Del model</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="Line" name="line_9">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_8" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>360</y>
     <width>771</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="1" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Log:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>372</width>
         <height>212</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTextBrowser" name="log_area"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Algorithm output results:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>371</width>
         <height>212</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTextBrowser" name="algo_area"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="Line" name="line_10">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_12" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>761</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,1,1,1,2,1">
    <item>
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Support:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="browserlabel" name="web_label1">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;OpenVINO Introduction&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="browserlabel" name="web_label3">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;Get online supported&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="version_label">
      <property name="text">
       <string>Version:1.1.5</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>roi_label</class>
   <extends>QLabel</extends>
   <header location="global">my_class/roi_label.h</header>
  </customwidget>
  <customwidget>
   <class>browserlabel</class>
   <extends>QLabel</extends>
   <header location="global">my_class/browserlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>model_1st</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>Widget</receiver>
   <slot>model_1st_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>model_2nd</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>Widget</receiver>
   <slot>model_2nd_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yuv_stream</sender>
   <signal>clicked(bool)</signal>
   <receiver>Widget</receiver>
   <slot>yuv_selected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>h264_stream</sender>
   <signal>clicked(bool)</signal>
   <receiver>Widget</receiver>
   <slot>h264_selected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mjpeg_stream</sender>
   <signal>clicked(bool)</signal>
   <receiver>Widget</receiver>
   <slot>mjpeg_selected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>min_score_spinbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>min_score_value</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>min_score_value</sender>
   <signal>valueChanged(int)</signal>
   <receiver>min_score_spinbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scale_spinbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>scale_value</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>525</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>593</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scale_value</sender>
   <signal>valueChanged(int)</signal>
   <receiver>scale_spinbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>653</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>513</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>model_1st_changed()</slot>
  <slot>model_2nd_changed()</slot>
  <slot>h264_selected()</slot>
  <slot>mjpeg_selected()</slot>
  <slot>h265_selected()</slot>
  <slot>yuv_selected()</slot>
 </slots>
</ui>
